plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -1,60 +1,39 @@
 package io.jenkins.plugins.casc;
 
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonParser;
 import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
 import io.jenkins.plugins.casc.misc.Util;
-import org.everit.json.schema.Schema;
-import org.everit.json.schema.ValidationException;
-import org.everit.json.schema.loader.SchemaLoader;
 import org.json.JSONObject;
 import org.json.JSONTokener;
 import org.junit.Rule;
 import org.junit.Test;
 
-import static io.jenkins.plugins.casc.SchemaGeneration.generateSchema;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class SchemaGenerationTest {
+
     @Rule
     public JenkinsConfiguredWithCodeRule j = new JenkinsConfiguredWithCodeRule();
 
     @Test
-    public void validSchemaShouldSucceed() throws Exception {
-        JSONObject schemaObject = generateSchema();
-        JSONObject jsonSchema = new JSONObject(
-            new JSONTokener(schemaObject.toString()));
-        String yamlStringContents = Util.toStringFromYamlFile(this, "validSchemaConfig.yml");
-        JSONObject jsonSubject = new JSONObject(
-            new JSONTokener(Util.convertToJson(yamlStringContents)));
-        Schema schema = SchemaLoader.load(jsonSchema);
-        try {
-            schema.validate(jsonSubject);
-        } catch (Exception e) {
-            fail(e.getMessage());
-        }
+    public void validSchemaShouldSucceed() {
+        assertTrue(Util.validateSchema(returnJSONSubject("validSchemaConfig.yml")));
     }
 
     @Test
-    public void invalidSchemaShouldNotSucceed() throws Exception {
-        JSONObject schemaObject = generateSchema();
-        Gson gson = new GsonBuilder().setPrettyPrinting().create();
-        JsonParser jsonParser = new JsonParser();
-        JsonElement jsonElement = jsonParser.parse(schemaObject.toString());
-        String prettyJsonString = gson.toJson(jsonElement);
-        JSONObject jsonSchema = new JSONObject(
-            new JSONTokener(prettyJsonString));
-        String yamlStringContents = Util.toStringFromYamlFile(this, "invalidSchemaConfig.yml");
-        JSONObject jsonSubject = new JSONObject(
-            new JSONTokener(Util.convertToJson(yamlStringContents)));
-        Schema schema = SchemaLoader.load(jsonSchema);
+    public void invalidSchemaShouldNotSucceed() {
+        assertFalse(Util.validateSchema(returnJSONSubject("invalidSchemaConfig.yml")));
+
+    }
+
+    private JSONObject returnJSONSubject(String yamlFile)  {
+        String yamlStringContents = null;
         try {
-            schema.validate(jsonSubject);
-            fail();
-        } catch (ValidationException ve) {
-            ve.printStackTrace();
+            yamlStringContents = Util.toStringFromYamlFile(this, yamlFile);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Please avoid doing so in modern test tools or the production code. System logging is your friend

Reply:
Yeah added the System Logging